src/shorthands/JBox.jl

```diff
@@ -0,0 +1,53 @@
+function _JBox(cornerpoint1::Point, cornerpoint2::Point, color::String, action::Symbol, vertices::Bool)
+    sethue(color)
+    box(cornerpoint1, cornerpoint2, action, vertices=vertices)
+end
+function _JBox(points::Array, color::String, action::Symbol, vertices::Bool)
+    sethue(color)
+    box(points, action, vertices=vertices)
+end
+function _JBox(pt::Point, width::Real, height::Real, color::String, action::Symbol, vertices::Bool)
+    sethue(color)
+    box(pt, width, height, action, vertices=vertices)
+end
+function _JBox(pt::Point, width::Real, height::Real, cornerradius::Float64, color::String, action::Symbol)
+    sethue(color)
+    box(pt, width, height, cornerradius, action=action)
+end
+
+"""
+    JBox(cornerpoint1::Point, cornerpoint2::Point; color="black", action=:stroke, vertices=false)
+
+Create a box (rectangle) between two points and do an action.
+Use vertices=true to return an array of the four corner points: bottom left, top left, top right, bottom right.
+"""
+JBox(cornerpoint1::Point, cornerpoint2::Point; color="black", action=:stroke, vertices=false) = (args...) -> _JBox(cornerpoint1, cornerpoint2, color, action, vertices)
+
+"""
+    JBox(points::Array; color="black", action=:stroke, vertices=false)
+
+Create a box/rectangle using the first two points of an array of Points to defined opposite corners.
+Use vertices=true to return an array of the four corner points: bottom left, top left, top right, bottom right.
```

Comment:
Regarding: #141 we might want to return the upper left corner by default?

Reply:
Do we need to compute that? Because there's no direct way of  doing that?